src/deluge/gui/ui/browser/sample_browser.cpp

```diff
@@ -468,6 +471,14 @@ ActionResult SampleBrowser::buttonAction(deluge::hid::Button b, bool on, bool in
 		}
 	}
 
+	// Learn button: toggle autoload (only for non-audioclips)
+	else if (b == LOAD && getCurrentClip()->type != ClipType::AUDIO) {
+		if (!on) {
```

Comment:
was "(!on)" intentional here? typically I've seen (on)

Reply:
Yes, so it works on release, for consistency, just like with toggling scale mode in clips for example